chevah_build

```diff
@@ -880,11 +875,11 @@ command_test_compat() {
     execute rm -rf compat
     execute git clone https://github.com/chevah/compat.git --depth=1 -b master
     execute pushd compat
+    # Copy over the latest paver stuff, as some changes might require it.
+    execute cp ../../paver.{conf,sh} ./
```

Comment:
copying paver.conf is not quite ok... but for now I don't think there are issues.

compat, might have different base requirement, especially brink. so using a different version of brink might break the compat buiild system.

I think that we should copy just paver.sh and overwrite only PYTHON_CONFIGURATIO